README.md

```diff
@@ -45,6 +45,16 @@ should probably acquaint yourself with at https://github.com/TomPoulton/hiera-ey
 Note that in order to install the gpgme gem you'll need to have the ruby development package installed
 for your distribution.
 
+For use on puppetserver
+---
+
+As root
+
+```sh
+/opt/puppetlabs/puppet/bin/gem install hiera-eyaml-gpg ruby_gpg
```

Comment:
This is optional??  Perhaps just needed for `puppet lookup`?

Reply:
I would suspect so.  I did `/opt/puppetlabs/puppet/bin/gem uninstall hiera-eyaml-gpg ruby_gpg` in my Puppet master and it still merrily decrypts my values.